lib/clear.js

```diff
@@ -0,0 +1,169 @@
+import {
+    client,
+    categoriesService,
+    ordersService,
+    stateService,
+    customersService,
+    shippingMethodService,
+    zonesService,
+    productsService
+} from './services'
+
+const deleteCategories = () => {
+    client.execute({
+        uri: categoriesService.build(),
+        method: 'GET',
+    }).then(result => {
+            return Promise.all(
+                    result.body.results.map((element) => {
```

Comment:
Is map executed in parallel or linearly iterating the array?   

Linear would be very slow (everyhign is a remote HTTP call).  You ideal code pattern would make sure that the client you are using has the `queue` middleware in it ( https://commercetools.github.io/nodejs/sdk/api/sdkMiddlewareQueue.html ) and your code is triggering all the calls in parallel until all promises are completed. 

So e.g. if you're calling process() with a call that is fetching 100 per page and have a maximum concurrency of e.g. 20 in the `queue` middleware you could be pretty fast.

Reply:
I see your point, I will try to use `queue` here with `process()` to make it quicker